src/Tools/ExternalAccess/LiveShare/Completion/RoslynCompletionProvider.Exports.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.Completion;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.LiveShare.Completion
+{
+    [ExportCompletionProvider("CSharpLspCompletionProvider", StringConstants.CSharpLspLanguageName), Shared]
+    internal class CSharpLspCompletionProvider : RoslynCompletionProvider
+    {
```

Comment:
same typescript.

Reply:
ah. just remembered that typescript actually use VSSDK so they don't need it. sorry. ignore.